src/internal/ppsutil/util.go

```diff
@@ -436,7 +436,7 @@ func GetStatsCommit(commitInfo *pfs.CommitInfo) *pfs.Commit {
 			return commitRange.Lower
 		}
 	}
-	// TODO: Getting here would be a bug in 2.0, log?
+	// Returns nil in the case our Commit is in Spout or Service repo
```

Comment:
@PFedak before I merge, does this comment use the right terminology? 

Is it that the commit is being processed by Spout or Service pipeline? I'm not entirely sure of the commit semantics around those cases

Reply:
I'd say in the case it's an output commit for a spout or service pipeline (those are really properties of the pipeline rather than the repo), specifically because we don't enable stats in those cases.